### PR TITLE
[www] features expanded

### PR DESCRIPTION
**Pull request by josvanos**
## 🎯 Changes

> "prepare for trouble, make it double " (+1 if you know the reference) 

So a week ago I noticed that the features section was consisting of 3 features. I am convinced that tRPC has more that 3 features (advantages). What I really noticed was that the mentioned features contained multiple features in it self. For example auto type safety also contained auto completion. Light & snappy are two features. Batteries included was also merged with Framework agnostic. 

So this pull requests consist of 6 features spreading them out to go deeper in on the features of tRPC.  

Note: Please change it to your liking, as you guys are far more aware of the features of tRPC and which features you as tRPC want to highlight  
Note: Please check grammar ect. 

I no I did not make a  pre announcement that I would edit this, so do with this pull request whatever you want

## ✅ Checklist

- [ ] I have followed the steps listed in the [Contributing guide](https://github.com/trpc/trpc/blob/main/CONTRIBUTING.md).
- [ ] If necessary, I have added documentation related to the changes made.
- [ ] I have added or updated the tests related to the changes made.

### CODE REVIEW

**Review by jlalmes — APPROVED** [review]
❤️

**Review by bautistaaa — APPROVED** [review]

File: www/src/components/Features.tsx [hunk 1]
```diff
@@ -1,30 +1,53 @@
...
     icon: <FiLock size={20} />,
     color:
       'dark:bg-indigo-900/50 bg-indigo-200 dark:text-indigo-300 text-indigo-600',
   },
   {
-    title: 'Light & Snappy DX',
+    title: 'Snappy DX',
     description:
-      'No code generation, run-time bloat, or build pipeline. Zero dependencies & a tiny client-side footprint.',
+      'tRPC has no building or compiling steps, meaning no code generation, run-time bloat, or build pipeline. ',
```
Comment (juliusmarminge) [inline www/src/components/Features.tsx:17]: Is this grammatically correct?

```suggestion
      'tRPC has no build or compile steps, meaning no code generation, runtime bloat, or build pipeline.',

File: www/src/components/Features.tsx [hunk 2]
```diff
@@ -1,30 +1,53 @@
...
-import { FiBriefcase, FiLock, FiZap } from 'react-icons/fi';
+import { FaBatteryFull, FaSeedling } from 'react-icons/fa';
+import { FiBriefcase, FiLock, FiTerminal, FiZap } from 'react-icons/fi';
 
 const features = [
   {
     title: 'Automatic typesafety',
     description:
-      'Automatic typesafety & autocompletion inferred from your API-paths, their input data, & outputs.',
+      'Made a server side change? Typescript will warn you client side! This is inferred from your API-paths, their inputs & outputs.',
```
Comment (juliusmarminge) [inline www/src/components/Features.tsx:9]:
```suggestion
      'Made a server-side change? Typescript will immediately throw errors on the client before you even save the file! This is inferred from your API-paths, their inputs & outputs.',
```

maybe "immediately throw errors" is a bit aggressive 🤔

File: www/src/components/Features.tsx [hunk 3]
```diff
@@ -1,30 +1,53 @@
...
     icon: <FiZap size={20} />,
     color:
       'dark:bg-amber-900/50 bg-amber-200 dark:text-amber-300 text-amber-600',
   },
   {
-    title: 'Add to existing brownfield project',
+    title: 'Add easily to any web framework',
     description:
-      'Easy to add to your existing brownfield project with adapters for Connect/Express/Next.js.',
+      "Compatible with every javascript framework, such as React, Vue or Svelte. It's easy to add to your existing brownfield project. ",
```
Comment (juliusmarminge) [inline www/src/components/Features.tsx:25]:
```suggestion
      'Compatible with every JavaScript framework, such as React, Vue or Svelte. It's easy to add to your existing brownfield project.',

File: www/src/components/Features.tsx [hunk 4]
```diff
@@ -1,30 +1,53 @@
...
+    title: 'Light bundle size',
+    description:
+      'tRPC has zero dependencies and a tiny client-side footprint. ',
+    icon: <FaSeedling size={20} />,
+    color: 'dark:bg-lime-900/50 bg-lime-200 dark:lime-orange-300 text-lime-600',
+  },
+  {
+    title: 'Batteries included',
+    description:
+      'We provide adapters for React / Next.js / Express.js. Community maintained adapters are available for frameworks as Nuxt and Sveltekit.   ',
```
Comment (juliusmarminge) [inline www/src/components/Features.tsx:47]:
```suggestion
      'We provide adapters for React, Next.js, Express.js and more. There are also community maintained adapter available for frameworks such as Nuxt and Sveltekit.',

**Comment by juliusmarminge** [comment]
Didn't push directly cause its Friday afternoon and I'm tired so might not be correct on these suggestions 😸

File: www/src/components/Features.tsx (hunk 2 above)
Comment (bautistaaa) [reply www/src/components/Features.tsx:9]: how about

`Instant feedback with type errors`

Comment (KATT) [reply www/src/components/Features.tsx:9]:
```suggestion
      'Made a server-side change? Typescript will instantly show red squiggly lines on the client before you even save the file! This is inferred from your API-paths, their inputs & outputs.',
```

File: www/src/components/Features.tsx (hunk 3 above)
Comment (KATT) [reply www/src/components/Features.tsx:25]:
```suggestion
      "Compatible with any JavaScript framework, such as React, Vue or Svelte. It's easy to add to your existing brownfield project. ",
```

File: www/src/components/Features.tsx (hunk 4 above)
Comment (KATT) [reply www/src/components/Features.tsx:47]:
```suggestion
      'We provide adapters for React, Next.js, Express.js, and more. There are also community-maintained adapters available for frameworks such as Nuxt and Sveltekit.',
```

**Comment by KATT** [comment]
YOLO. This is great. Thanks so much @josvanos & please setup github sponsors so I can buy you some drinks this weekend

**Pull request merged**